doc-examples/jdbc-example-java/src/main/java/example/UserRepositoryWithWhere.java

```diff
@@ -0,0 +1,13 @@
+package example;
+
+import io.micronaut.data.annotation.Where;
+
+import java.util.List;
+
+public interface UserRepositoryWithWhere {
+
+    // ...
+
+    @Where("@.enabled = false")
```

Comment:
Did you want to have `@IgnoreWhere` here?

Reply:
No, it's just an example of using @Where in a repository method, since the documentation hasn't included one yet.

Reply:
Ah ok